card/mod/pointer/set/abstract/00_paging_params.rb

```diff
@@ -1,3 +1,5 @@
+MAX_ANONYMOUS_SEARCH_PARAM = 1000
```

Comment:
This was an emergency patch that I put out last night after seeing that there were a lot of requests with extremely high offsets that were taking a super long time to complete.  Not a durable solution.